src/azure-cli/azure/cli/command_modules/containerapp/_client_factory.py

```diff
@@ -0,0 +1,111 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# pylint: disable=line-too-long, consider-using-f-string
+
+from azure.cli.core.commands.client_factory import get_mgmt_service_client
+from azure.cli.core.profiles import ResourceType
+from azure.cli.core.azclierror import CLIInternalError
+
+
+# pylint: disable=inconsistent-return-statements
+def ex_handler_factory(no_throw=False):
+    def _polish_bad_errors(ex):
+        import json
+        try:
+            content = json.loads(ex.response.content)
+            if 'message' in content:
+                detail = content['message']
+            elif 'Message' in content:
+                detail = content['Message']
+
+            ex = CLIInternalError(detail)
+        except Exception:  # pylint: disable=broad-except
+            pass
+        if no_throw:
+            return ex
+        raise ex
+    return _polish_bad_errors
+
+
+def handle_raw_exception(e):
+    import json
+
+    stringErr = str(e)
+
+    if "WorkloadProfileNameRequired" in stringErr:
+        raise CLIInternalError("Workload profile name is required. Please provide --workload-profile-name.")
+
+    if "Unknown properties Name in Microsoft.ContainerApps.WebApi.Views.Version20221101Preview.WorkloadProfile are not supported" in stringErr:
```

Comment:
which scenario the error message will return?

Reply:
deprecate code, removed.